src/actions-on-google.ts

```diff
@@ -155,6 +155,9 @@ interface ActionResponseItem extends JsonObject {
             icon: {
                 url: string,
             },
+            largeImage: {
```

Comment:
Should this be flagged as optional too?

Reply:
@atulep Actually, any errors occurs. Instead, if the icon or largeImage is not specified, `icon: undefined` or `largeImage: undefined` will be displayed. But, I think that it would be a good that any output is not shown if the value is undefined. I will fix this. Thank you for the advice.